common/src/main/scala/org/apache/celeborn/common/meta/WorkerInfo.scala

```diff
@@ -126,13 +147,19 @@ class WorkerInfo(
     }
   }
 
-  def readableAddress(): String = {
-    s"Host:$host:RpcPort:$rpcPort:PushPort:$pushPort:" +
-      s"FetchPort:$fetchPort:ReplicatePort:$replicatePort"
+  def readableAddress(authEnabled: Boolean): String = {
+    if (authEnabled) {
+      s"Host:$host:RpcPort:$rpcPort:PushPort:$pushPort:" +
+        s"FetchPort:$fetchPort:ReplicatePort:$replicatePort:InternalPort:$internalPort" +
+        s":SecuredRpcPort:$securedRpcPort:SecuredPushPort:$securedPushPort:SecuredFetchPort:$securedFetchPort"
+    } else {
+      s"Host:$host:RpcPort:$rpcPort:PushPort:$pushPort:" +
+        s"FetchPort:$fetchPort:ReplicatePort:$replicatePort:InternalPort:$internalPort"
+    }
```

Comment:
This could be simplified into the following:
```
val addressStr = s"Host:$host:RpcPort:$rpcPort:PushPort:$pushPort:" +
        s"FetchPort:$fetchPort:ReplicatePort:$replicatePort:InternalPort:$internalPort"
if (authEnabled) {
    addressStr +=  s":SecuredRpcPort:$securedRpcPort:SecuredPushPort:$securedPushPort:SecuredFetchPort:$securedFetchPort"
}
addressStr
```